front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/CommunityMonitorCTA/index.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+
+import {
+  Box,
+  Button,
+  Icon,
+  Title,
+  Image,
+  Text,
+  useBreakpoint,
+} from '@citizenlab/cl2-component-library';
+import { useTheme } from 'styled-components';
+import { Multiloc } from 'typings';
+
+import useCommunityMonitorProject from 'api/community_monitor/useCommunityMonitorProject';
+import usePhase from 'api/phases/usePhase';
+
+import useInputSchema from 'hooks/useInputSchema';
+import useLocalize from 'hooks/useLocalize';
+
+import { useIntl } from 'utils/cl-intl';
+import clHistory from 'utils/cl-router/history';
+import { calculateEstimatedSurveyTime } from 'utils/surveyUtils';
+
+import SentimentQuestionPreview from './assets/SentimentQuestionPreview.png';
+import messages from './messages';
+import Settings from './Settings';
+interface Props {
+  title: Multiloc;
+  description: Multiloc;
+  surveyButtonText: Multiloc;
+}
+
+const CommunityMonitorCTA = ({
+  title,
+  description,
+  surveyButtonText,
+}: Props) => {
+  const theme = useTheme();
+  const localize = useLocalize();
+  const { formatMessage } = useIntl();
+
+  const isTabletOrSmaller = useBreakpoint('tablet');
+  const isMobileOrSmaller = useBreakpoint('phone');
+
+  const { data: project } = useCommunityMonitorProject({});
+  const phaseId = project?.data.relationships.current_phase?.data?.id;
+
+  // Get the community monitor phase & check if the survey is currently open
+  const { data: phase } = usePhase(phaseId);
+
+  const isSurveyLive = phase?.data.attributes.submission_enabled;
+
+  // Get the survey UI schema
+  const { uiSchema } = useInputSchema({
+    projectId: project?.data.id || '',
+    phaseId,
+  });
+
+  // Calculate estimated time to complete survey
+  const estimatedMinutesToComplete = calculateEstimatedSurveyTime(uiSchema);
+
+  const goToCommunityMonitorSurvey = () => {
+    if (phaseId) {
+      clHistory.push(
+        `/projects/${project.data.attributes.slug}/surveys/new?phase_id=${phaseId}`
+      );
+    }
+  };
+
+  if (!isSurveyLive) {
+    return null;
+  }
+
+  return (
+    <Box
+      mx="auto"
+      w="100%"
+      maxWidth="1200px"
+      px={isTabletOrSmaller ? '16px' : '0'}
+    >
+      <Box
+        background={theme.colors.tenantPrimaryLighten90}
+        w="100%"
+        borderRadius="16px"
+        p="32px"
+        display="flex"
+        flexDirection={isTabletOrSmaller ? 'column' : 'row'}
+        gap="16px"
+      >
+        <Box maxWidth={isTabletOrSmaller ? '100%' : '40%'}>
+          <Title mt="0px" color="grey800" variant="h2" fontWeight="semi-bold">
```

Comment:
Could we remove the `fontWeight="semi-bold"`? I spent a lot of time in the past making all our titles consistent, so I would really prefer to not have any deviating font weights anymore unless it's really an exceptional situation